app/controllers/habits_controller.rb

```diff
@@ -0,0 +1,53 @@
+class HabitsController < ApplicationController
+  before_action :authorize_request
```

Comment:
I would put this in the base API controller and `skip_before_action` any routes you don't want to authorize. That will clean this up and you don't have to remember to put it in each controller.

Reply:
Good idea!